packages/ember-routing/lib/location/history_location.js

```diff
@@ -288,7 +287,9 @@ export default EmberObject.extend({
 
     @method getHash
   */
-  getHash: EmberLocation._getHash,
+  getHash() {
```

Comment:
`getHash,` should work?

Reply:
`utils/getHash` expects location object, while this `class getHash` does not expect any arguments